EIPS/eip-7069.md

```diff
@@ -63,21 +63,22 @@ Execution semantics of `EXT*CALL`:
 3. If `value` is non-zero:
     - Halt with exceptional failure if the current frame is in `static-mode`.
     - Charge `CALL_VALUE_COST` gas.
-4. Peform (and charge for) memory expansion using `[input_offset, input_size]`.
-5. If `target_address` is not in the `warm_account_list`, charge `COLD_ACCOUNT_ACCESS - WARM_STORAGE_READ_COST` (2500) gas.
-6. If `target_address` is not in the state and the call configuration would result in account creation, charge `ACCOUNT_CREATION_COST` (25000) gas.
+4. Halt with exception if any of the high 96 bits of the `target_address` are set.
```

Comment:
There is an option to leave EIP-7069 intact, and put this step as an expansion of EXT*CALL logic into the new EIP-7676. The advantage is decoupling EIPs and not leaving this step without its due motivation described in EIP-7069 (or worse, having to reference circularly the EIP-7676).

But I'm not strongly against laying this out like is done now.

Reply:
It's the scattering of call across only two EIPs that concerns me.  For vanilla validation it's already across multiple, and the rules focus on the operations in those EIPs.  But splitting operational steps across EIPs I am concerned will result in implementations overlooking the detail.